### ISSUE TITLE
[BUG] Both default minecraft event messages and player events messages display in chat

### ISSUE THREAD

**Issue by ClaudeGontier** [ByMartrixx/player-events#15]
**Describe the bug**
Both default minecraft event messages and player events messages display in chat

**To Reproduce**
Steps to reproduce the behaviour:
1. Install Player Events 2.2.2 (1.18)
2. Config the mod's messages
3. Both the mod's messages and vanilla messages display

**Expected behaviour**
Only the mod's messages should display

![Capture](https://user-images.githubusercontent.com/54834036/145589657-703f1beb-a57d-4e02-a22f-367bb2811af5.PNG)

**Comment by IotaBread** [comment]
Currently, this is intended behavior. In some release I'll add an option to cancel/override the vanilla message

**Comment by zachthedev** [comment]
@ByMartrixx Has this been implemented yet?

**Comment by IotaBread** [comment]
No, sorry. I forgot about this and lately I've been quite busy, so it may take me a while to finally implement this
